src/main/java/dev/dkvl/womutils/panel/WomPanel.java

```diff
@@ -363,40 +272,59 @@ else if (rawString.equalsIgnoreCase("last updated"))
 
     private void applyOverviewResult(PlayerInfo result)
     {
-        for (InfoLabel infoLabel : miscInfoLabels)
+        for (MiscInfo infoLabel : miscInfoLabels)
         {
             JLabel label = infoLabel.getLabel();
-            switch (infoLabel.getRawString().toLowerCase())
+            switch (infoLabel)
             {
-                case "country":
+                case COUNTRY:
```

Comment:
The formatting should be provided by overriding a format function for each MiscInfo enum entry instead of this switch statement. This can then be used in the previously mentioned wrapper object to format the label.